.circleci/config.yml

```diff
@@ -14,7 +14,7 @@ parameters:
     # 3. Commit this change to the PR branch where the changes exist.
     current_golden_images_hash:
         type: string
-        default: 9995fe6e5347d0dfd21b6ef153068c4004fb1987
+        default: 6e76c8fd1c26c6b191215ba4a405ec3149c85b46
```

Comment:
Obvi, since this is a fix to styling issues, the current hash needed to be updated